custom_components/google_home/const.py

```diff
@@ -64,7 +64,7 @@
     HEADER_CAST_LOCAL_AUTH: "",
     HEADER_CONTENT_TYPE: "application/json",
 }
-TIMEOUT = 10  # Request Timeout in seconds
+TIMEOUT = 5  # Request Timeout in seconds
```

Comment:
Let's set it to 2 seconds. In my case all requests are completed in 200ms, that should be more than enough.

Reply:
Changed it to 2 seconds. 😄